dtfs-central-api/api-tests/mocks/gef/gef-applications.js

```diff
@@ -0,0 +1,230 @@
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 8',
+  additionalRefName: 'Team 8',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 9',
+  additionalRefName: 'Team 9',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 10',
+  additionalRefName: 'Team 10',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 11',
+  additionalRefName: 'Team 11',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 12',
+  additionalRefName: 'Team 12',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 13',
+  additionalRefName: 'Team 13',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 14',
+  additionalRefName: 'Team 14',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+}, {
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+  maker: {},
+  bank: {},
+  bankInternalRefName: 'Bank 15',
+  additionalRefName: 'Team 15',
+  exporter: {},
+  createdAt: '2021-01-01T00:00',
+  mandatoryVersionId: '123',
+  status: CONSTANTS.DEALS.DEAL_STATUS.IN_PROGRESS,
+  updatedAt: null,
+  submissionCount: 0,
+  editedBy: [null],
+  facilitiesUpdated: null,
+},
+{
+  dealType: 'GEF',
```

Comment:
constant

Reply:
done

Reply:
I can't see this change in the code. are you sure you pushed it?

Reply:
yes, this file is now only 29 lines long 🤔